src/GraphQLParser.Tests/Files/DefinitionSortTests.graphql

```diff
@@ -0,0 +1,31 @@
+scalar Type3
+
+schema {
+  mutation: Type2
+  query: Type3
```

Comment:
@Shane32 Hi! My break from GraphQL was longing for a much more time than I thought. I changed the company and had to move away for some time.

> A schema definition will have the query first, mutation second, and subscription defined third

Somethig wrong here.

Reply:
@sungam3r welcome back !

What's wrong?

Reply:
Oh!  Yup.

Reply:
This is the input file.  The sorted output is in `SDLSorterTests.SortsSchemaDefinition.approved.DefinitionSortTests.graphql`

Reply:
Oh, it's a source file 🤦‍♂️ .